src/status_im/extensions/capacities/components.cljs

```diff
@@ -23,9 +25,10 @@
     (js/clearTimeout id))
   (reset! current (js/setTimeout #(on-input-change-text on-change value) delay)))
 
-(defn input [{:keys [keyboard-type style on-change change-delay placeholder placeholder-text-color selection-color
+(defn input [{:keys [keyboard-type style error on-change change-delay placeholder placeholder-text-color selection-color
```

Comment:
Is error surfaced in the capacity map?

Reply:
@jeluard sorry, I didn't get that, what do you mean?